develop-docs/sdk/telemetry/traces/span-links.mdx

```diff
@@ -0,0 +1,221 @@
+---
+title: Span Links
+---
+
+Span links associate one span with one or more spans. The most important application of linking traces are frontend applications. It's also useful in settings with producer/consumer patterns.
+We are able to display a user journey in our tracing views to make debugging of issues easier as developers get more context on what happened before a specific issue.
+Without somehow linking spans together, we are limited to the duration of one trace (id), which is handled differently and kept alive for different times across our various SDKs.
```

Comment:
l: I would omit this from the intro as this already is quite specific towards previous/next trace links. I'd rather keep the intro short and more general.

```suggestion
```